web/client/components/mapcontrols/locate/LocateTool.jsx

```diff
@@ -34,8 +46,13 @@ const LocateTool = ({map, mapType, status, messages, changeLocateState, onLocate
         changeLocateState("DISABLED");
     };
 
+    /**
+     * when loaded do something, use the start method
```

Comment:
Is this comment useful?

Reply:
i was thinking it could be useful for new comers but i get your point of no-comment at all 😝